disk-encryption-repart-dracut

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: Copyright 2024 SUSE LLC
+set -e
+shopt -s nullglob
+
+unset "${!LC_@}"
+LANG="C.utf8"
+export LANG
+
+blkdev=
+blkpart=
+mp=
+
+log_info()
+{
+	echo "$@"
+}
+
+err()
+{
+	echo "Error: $*" >&2
+	echo "Error: $*"
+	exit 1
+}
+
+####### main #######
+
+# silence systemd
+kill -SIGRTMIN+21 1
```

Comment:
If this is ment to write to stdout or such, using `StandardOutput=journal+console` should be enough

Reply:
Yes, that is meant. Great, that looks good !